openlibrary/templates/lib/nav_foot.html

```diff
@@ -47,6 +47,17 @@ <h2>$:_('Help')</h2>
           <a class="footer-icons__github" href="https://github.com/internetarchive/openlibrary">github</a>
         </aside>
       </div>
+      <div id="footer-locale-menu">
+        <h2>$:_('Change Website Language')</h2> 
```

Comment:
In the web.py templating system, `$` does HTML escaping, whereas `$:` does not. So `$` is preferred as the default unless we know we're going to have some HTML! (E.g. `$:_("<a>Foo</a>")`)

```suggestion
        <h2>$_('Change Website Language')</h2> 
```